package.json

```diff
@@ -20,13 +20,17 @@
     "start": "npm run test-studio",
     "ecommerce-studio": "gulp ecommerce-studio",
     "test-studio": "gulp test-studio",
+    "backstop-test-studio": "gulp backstop-test-studio",
```

Comment:
Is this script still used or can it be removed?

Reply:
Ah, it's used for updating content.

PS. Would be nice to document how to use these tests in the root README.